src/Extension/Extension.php

```diff
@@ -283,6 +283,14 @@ public function getIconStyles(): string
     {
         $properties = $this->getIcon();
 
+        if (empty($properties)) {
+            return '';
+        }
+
+        $properties = array_filter($properties, function ($item) {
+            return ! is_array($item);
```

Comment:
Would it be better to check with `is_string()`?

Reply:
oh definitely 🤦🏼 

```suggestion
            return is_string($item);
```